### ISSUE TITLE
Some questions on `inference_IMAGdressing_controlnetinpainting.py`

### ISSUE THREAD

**Issue by ipfans** [muzishen/IMAGDressing#7]
Thanks for your great work. I ran into some confusion when trying `inference_IMAGdressing_controlnetinpainting.py` file.

* The README mentions that we need to download the model from the `IDM-VTON` repo，but which model should be downloaded?
* `h94/IP-Adapter` does not contains a file named `config.json`. It will be failed to start the pipeline with: `OSError: h94/IP-Adapter does not appear to have a file named config.json. Checkout 'https://huggingface.co/h94/IP-Adapter/tree/main' for available files.`

**Comment by WhiteJiang** [comment]
Thank you for your attention. VTON's ckpt/humanparsing and ckpt/openpose need to be downloaded. For IP-Adapter, we used its models/image_encoder, and the corresponding config.json is also in this directory.

**Comment by ipfans** [comment]
@WhiteJiang thank you for your reply. For ip-adapter, I think we can add `subfolder="models/image _ encoder"` to avoid confusion.

**Comment by WhiteJiang** [comment]
@ipfans thanks for your advice

**Comment by ipfans** [comment]
@WhiteJiang also, this project deps seems break on this file: 

```
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/ubuntu/imagd/inference_IMAGdressing_controlnetinpainting.py", line 264, in <module>
    output = pipe(
  File "/home/ubuntu/imagd/.venv/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/ubuntu/imagd/dressing_sd/pipelines/IMAGDressing_v1_pipeline_controlnet_inpainting.py", line 262, in __call__
    control_image = self.prepare_control_image(
  File "/home/ubuntu/imagd/dressing_sd/pipelines/IMAGDressing_v1_pipeline_controlnet_inpainting.py", line 95, in prepare_control_image
    image = self.control_image_processor.preprocess(
TypeError: VaeImageProcessor.preprocess() got an unexpected keyword argument 'crops_coord
```

I am currently using version 0.24.0 of diffusers, which is the same as in the requirements.txt file。

```
pip list |grep diffusers
diffusers                0.24.0
```

**Comment by WhiteJiang** [comment]
@ipfans This is a bug caused by the diffuser version. We solved the problem after upgrading to version 0.26.3

**Comment by ipfans** [comment]
Oh, finally it works... thank you guys
